i18n/i18n.py

```diff
@@ -86,8 +86,8 @@ def validate_translation_files():
         sys.exit(1)
 
 
-def fill_missing():
```

Comment:
ok, got it.  the command is still "fill" but this script doesn't fill/alter translation files, rather it prints what is missing.